.depcheckrc.yml

```diff
@@ -22,6 +22,7 @@ ignores:
   - 'eslint-plugin-*'
   - 'jest-silent-reporter'
   - 'prettier-plugin-packagejson'
+  - 'prettier-plugin-tailwindcss'
```

Comment:
Adding `prettier-plugin-tailwindcss` prevents `Unused devDependencies` depcheck error which is comment for some of our dev dependencies 

<img width="340" alt="Screenshot 2025-04-05 at 12 15 46 PM" src="https://github.com/user-attachments/assets/d16f757e-12c5-4f29-827a-341f517ebf5f" />